### PR TITLE
Initial setup and porting of mockup

### PR DESCRIPTION
**Pull request by offtherailz**
With these changes the mock-up has been ported in a running web-app ready to be installed on georchestra.
The css is loaded inline. So basic structure is provided. 

The API is still missing (as for issue #7) but the base UI has been ported.

### CODE REVIEW

File: js/extensions/cadastrapp/components/Main.jsx [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+            setIsInformationFormShown(true);
+            break;
+        case "export":
+            alert("Selection export is clicked");
+            break;
+        case "zoom":
+            alert("Selection zoom is clicked");
+            break;
+        case "landry":
+            alert("You clicked on landry information button, you will redirecting an external service in a new tab");
```
Comment (landryb) [inline js/extensions/cadastrapp/components/Main.jsx:242]: Can you take https://github.com/georchestra/mapstore2-cadastrapp/issues/12#issuecomment-704301264 into account ? thanks !

File: js/extensions/cadastrapp/components/PlotSelection.jsx [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+            <OverlayTrigger placement="bottom" overlay={<Tooltip>{"Zoom"}</Tooltip>}>
+                <Button
+                    onClick={() => { props.onClick("zoom"); }}>
+                    <Glyphicon glyph="zoom-in" />
+                </Button>
+            </OverlayTrigger>
+            <OverlayTrigger placement="bottom" overlay={<Tooltip>{"Landry Information"}</Tooltip>}>
+                <Button
+                    {...(!isAtleastOneSelected ? { disabled: 'true' } : {})}
+                    onClick={() => { props.onClick("landry"); }}>
```
Comment (landryb) [inline js/extensions/cadastrapp/components/PlotSelection.jsx:164]: Can you take https://github.com/georchestra/mapstore2-cadastrapp/issues/12#issuecomment-704301264 into account for those two occurences ? thanks !

File: js/extensions/cadastrapp/components/Main.jsx (hunk 1 above)
Comment (tdipisa) [reply js/extensions/cadastrapp/components/Main.jsx:242]: @landryb don't worry this is just a first PR. We will address this comment when we will work on issue #12, as soon as we are informed about the new string to use.

Comment (landryb) [reply js/extensions/cadastrapp/components/Main.jsx:242]: well, i think we've settled on `unite_fonciere`, no ?

**Pull request merged**